docs/setting-up/preparing-pi.md

```diff
@@ -77,9 +76,14 @@
     7. Disable services that will interfere with the hotspots function with the following commands:
 
         ``` sh
+	sudo systemctl disable serial-getty@ttyAMA0.service && \
         sudo systemctl disable hciuart.service && \
         sudo systemctl disable bluealsa.service && \
-        sudo systemctl disable bluetooth.service
+        sudo systemctl disable bluetooth.service && \
+	sudo systemctl mask serial-getty@ttyAMA0.service && \
```

Comment:
Can you check indentation starting this line?  Seems to be one tab space off, which breaks the result

![image](https://github.com/Centrunk/centrunk.github.io/assets/38590447/86ac4e2c-9e36-4b62-94f7-8c489f786b47)

![image](https://github.com/Centrunk/centrunk.github.io/assets/38590447/f74d1d8a-afdc-4118-8207-29d4568034ae)